src/frontend/src/components/SearchPage/SearchPage.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import { useLazyQuery } from '@apollo/react-hooks';
+import { makeStyles } from '@material-ui/core/styles';
+import gql from 'graphql-tag';
+import Header from '../Header';
+import SearchBar from '../SearchBar';
+import AuthorResult from '../AuthorResult';
+
+const useStyles = makeStyles(() => ({
+  searchReply: {
+    margin: 'auto',
+    padding: '2rem',
+    color: 'coral',
+    fontWeight: 400,
+  },
+  divAfterHeader: {
+    height: '12vh',
```

Comment:
This will be huge on larger screens.  I would be careful with `vh` here.  Why not use flexbox or something?

Reply:
So just use `<Box display="flex"> </Box>` I tried this and my SearchBar disappeared

Reply:
The `vh` is just a bandaid solution for #765 . We don't have a solution to fix it yet. The SearchBar is actually underneath the Header bar